src/main/webapp/app/tasks/local-task.service.ts

```diff
@@ -32,6 +32,25 @@ export class LocalTaskService implements TaskService {
     return of(null);
   }
 
+  updateStatus(id: string, status: string): Observable<void> {
+    const tasks = this.readTasks();
+    const updated_tasks = tasks.map(task => {
+      if (task.id === id) {
+        task.status = status;
+      }
+      return task;
+    });
+    this.writeTasks(updated_tasks);
+    return of(null);
```

Comment:
why return observable of null?

Reply:
Updated. if api need to return the updated task . since we are using localstorage returned the full list.